### PR TITLE
added randomInt helper, added error route

### PR DESCRIPTION
**Pull request by Wyrd00**
relates #16, #17

### CODE REVIEW

File: src/views/helpers/randomInt.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = (req, res) => {
```
Comment (matthewdking) [inline src/views/helpers/randomInt.js]: I'm not sure about the input of req and res for this function

File: src/controllers/error.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+exports.client = (req, res) => {
+  res.status(404).render('error', {
+    layout: 'main',
+    statusCode: 404,
+    errorMessage: 'Experiencing difficulties - please come back and be judgemental later'
```
Comment (matthewdking) [inline src/controllers/error.js:5]: :laughing:

**Review by matthewdking — CHANGES_REQUESTED** [review]

File: src/views/helpers/randomInt.js (hunk 1 above)
Comment (Wyrd00) [reply src/views/helpers/randomInt.js]: good call

**Review by matthewdking — APPROVED** [review]

**Review by sajedazoabi — APPROVED** [review]

File: src/controllers/error.js (hunk 2 above)
Comment (Karyum) [reply src/controllers/error.js:5]: :+1:

File: src/controllers/error.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  })
+}
+
+exports.server = (req, res) => {
+  res.status(500).render('error', {
+    layout: 'main',
+    statusCode: 500,
+    errorMessage: 'Experiencing internal difficulties - express is hard, please don\'t judge us'
```
Comment (Karyum) [inline src/controllers/error.js:13]: lol

File: src/views/home.hbs [hunk 4]
```diff
@@ -1,4 +1,12 @@
 	<section class="main__container--limited-width">
 		<h1 class="main__title">HOME</h1>
 			<p>Welcome to Judgement Day, kids.</p>
+	</section>
+
+	<section class="main__container--full-width">
+		<ul class="mentors__face" id="mentors__face">
+			<li>
+				<p>hollabackgrrlll</p>
```
Comment (Karyum) [inline src/views/home.hbs:9]: damn sublime

File: src/views/layouts/error.hbs [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+
+<html>
+	{{>htmlHead}}
+	<body>
+		<main class="main">
```
Comment (Karyum) [inline src/views/layouts/error.hbs:6]: jesus..

**Review by Karyum — APPROVED** [review]
i hate sublime

**Pull request merged**